CONTRIBUTING.md

```diff
@@ -94,6 +94,8 @@ Language specific directories must follow the pattern `pages.<locale>`, where `<
 
 The `<country>` code is optional and should only be added when it is needed. In other words, only when there is a valid reason to distinguish between a language (`ll`) and its regional dialects (`ll_CC1`, `ll_CC2`, etc.). As an example, both `fr_FR` and `fr_BE` should fall under the same `pages.fr` directory, since there virtually is no difference in writing between standard French and Belgian French.
 
+To see the current progress of all translations, you can visit https://lukwebsforge.github.io/tldri18n/, which provides a dynamically updated table of all pages and their translations.
```

Comment:
What about the autolinking syntax here?
```suggestion
To see the current progress of all translations, you can visit <https://lukwebsforge.github.io/tldri18n/>, which provides a dynamically updated table of all pages and their translations.
```

Reply:
Hmm, I didn't know that was a think.  I guess I though we use it in our pages just for clarity :smile: